rdprail-shell/meson.build

```diff
@@ -35,6 +35,7 @@ if get_option('shell-rdprail')
 		dep_librsvg,
 		dep_winpr,
 		dep_glib,
+		dep_frdp_server,
```

Comment:
Ideally we don't need this if backend-rdp header doesn't include FreeRDP header.